opentracing-jaxrs2/src/main/java/io/opentracing/contrib/jaxrs2/server/ServerTracingDynamicFeature.java

```diff
@@ -73,10 +75,12 @@ protected String operationName(ResourceInfo resourceInfo) {
         private final Tracer tracer;
         private boolean allTraced;
         private List<ServerSpanDecorator> spanDecorators;
+        private int priority;
 
         private Builder(Tracer tracer) {
             this.tracer = tracer;
-            this.spanDecorators = Arrays.asList(ServerSpanDecorator.STANDARD_TAGS);
+            this.spanDecorators = Collections.singletonList(ServerSpanDecorator.STANDARD_TAGS);
+            this.priority = Priorities.HEADER_DECORATOR;
```

Comment:
I'd leave a comment here on why you decided for a `HEADER_DECORATOR`, specially because people might be tempted to send PRs changing this to `AUTHENTICATION`.